### PR TITLE
CLOCK CULT BUFF (GAMECHANGING)

### PR DESCRIPTION
**Pull request by Readystorm546**
So basically coggers on clock cult win but only on a 1% chance

:cl:  
rscadd: Roundend clock cult win Coggers
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
speedmerge this shit

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -242,7 +242,10 @@ Credit where due:
 	..()
 	if(GLOB.clockwork_gateway_activated)
 		SSticker.news_report = CLOCK_SUMMON
-		SSticker.mode_result = "win - servants completed their objective (summon ratvar)"
+		if(prob(99))
+			SSticker.mode_result = "win - servants completed their objective (summon ratvar)"
+		else
+			SSticker.mode_result = "Coggers"
```
Comment (JamieD1) [inline code/game/gamemodes/clock_cult/clock_cult.dm]:
```suggestion
		SSticker.mode_result = "win - servants completed their objective (summon ratvar)"
		if(prob(1))
			SSticker.mode_result = "Coggers"
```

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 2]
```diff
@@ -243,6 +243,8 @@ Credit where due:
 	if(GLOB.clockwork_gateway_activated)
 		SSticker.news_report = CLOCK_SUMMON
 		SSticker.mode_result = "win - servants completed their objective (summon ratvar)"
+		if(prob(1))
+			SSticker.mode_result = "Coggers"
```
Comment (JamieD1) [inline code/game/gamemodes/clock_cult/clock_cult.dm]:
```suggestion
		SSticker.mode_result = ( prob(1) ? "Coggers" : "win - servants completed their objective (summon ratvar)")
```

Comment (Hopekz) [reply code/game/gamemodes/clock_cult/clock_cult.dm]: cursed

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**